workspaces/quay/plugins/quay-actions/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@backstage-community/plugin-scaffolder-backend-module-quay",
+  "description": "The quay-actions module for @backstage-community/plugin-scaffolder-backend-module-quay",
+  "version": "2.2.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin-module",
+    "supported-versions": "1.32.5",
+    "pluginId": "scaffolder",
+    "pluginPackage": "@backstage/plugin-scaffolder-backend"
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint:check": "backstage-cli package lint",
+    "lint:fix": "backstage-cli package lint --fix",
+    "postpack": "backstage-cli package postpack",
+    "prepack": "backstage-cli package prepack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "tsc": "tsc",
+    "prettier:check": "prettier --ignore-unknown --check .",
+    "prettier:fix": "prettier --ignore-unknown --write ."
+  },
+  "dependencies": {
+    "@backstage/backend-plugin-api": "^1.0.1",
+    "@backstage/plugin-scaffolder-node": "^0.5.0",
+    "yaml": "^2.6.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.28.2",
+    "@backstage/plugin-scaffolder-node-test-utils": "0.1.14",
+    "@spotify/prettier-config": "^15.0.0",
+    "prettier": "3.3.3"
+  },
+  "files": [
+    "dist",
+    "dist-dynamic/*.*",
+    "dist-dynamic/dist/**"
```

Comment:
Should this be removed since we don't have the export-dynamic command in the plugin anymore?